modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/layouts/ActivationKeysLayout/Inputs/index.js

```diff
@@ -213,7 +219,9 @@ const ActivationKeysInputs = ({
 									key={
 										accountSubscription.accountSubscriptionId
 									}
-									label={accountSubscription.name}
+									label={i18n.translate(
+										accountSubscription.name
+									)}
```

Comment:
Should be good use the getKebab function

Reply:
Nice, done!!